docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -528,6 +528,153 @@ Required ECS fields when not using {beats}:::
 
 // end::siem-auditbeat-auth-jobs[]
 
+[discrete]
+[[security-authentication]]
+== Security: Authentication
+
+Detect anomalous activity in your ECS-compatible authentication logs.
+
+These configurations are only available if data exists that matches the 
```

Comment:
@spong - is this correct? Does the security app run the query in the `manifest.json` file to decide whether the displays the job configurations inside the security app?

Reply:
Since this content is used in both the ML docs and the Security docs, if this statement only applies to the ML app we can qualify it as such.

Reply:
@peteharverson -- we don't run the query from the manifest, but we do hit the `recognize` API providing the configured `securitySolution:defaultIndex` index pattern from Kibana Advanced Settings. Does this result in the same behavior as running the query specified in the `manifest.json`?

https://github.com/elastic/kibana/blob/4584a8b570402aa07832cf3e5b520e5d2cfa7166/x-pack/plugins/security_solution/public/common/components/ml_popover/api.ts#L23-L42

Reply:
Yes, hitting the `recognize` API with the configured index pattern results in the same behavior as running the query specified in the `manifest.json` file. But @lcawl do you think it's worth clarifying how the check in the security app uses the index pattern configured under `securitySolution:defaultIndex` from Kibana Advance Settings?

Something like

```
These configurations are only available if data in the securitySolution:defaultIndex index pattern configured under Kibana Advanced Settings exists that matches the query specified in the manifest file.
```

Reply:
To cover both circumstances, how about something like this:

> In the Machine Learning app, these configurations are available only when data exists
that matches the query specified in the [link to manifest file]. In the Elastic Security app, it looks
in the data view specified in the [securitySolution:defaultIndex advanced setting] for data that matches the query.

Reply:
Suggested text LGTM

Reply:
Thanks, I've added that info in e4cade7d5a93b5251aed9ff44dc91691ff5f3c3b